internal/testing/enginetest/enginetest.go

```diff
@@ -57,6 +59,60 @@ func RunTestEngine_NewModuleEngine(t *testing.T, et EngineTester) {
 	})
 }
 
+func RunTestEngine_InitializeFuncrefGlobals(t *testing.T, et EngineTester) {
```

Comment:
Maybe one for externref?

Reply:
Externref are not engine specific so no need for that!